unit-tests/func/presets/auto/ctrols-change/test-values.cpp

```diff
@@ -0,0 +1,55 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+//#cmake: static!
+
+#include "../../../func-common.h"
+#include "../../presets-common.h"
+#include <l500/l500-options.h>
+
+using namespace rs2;
+
+// This test checks that after changing one of the hw control, current and default values 
+// of all the others controls are correct
```

Comment:
are correct --> have not changed
right?

Reply:
yes